### ISSUE TITLE
Uncaught SyntaxError: Unexpected token } in JSON at position 287

### ISSUE THREAD

**Issue by TheNowak** [atom-community/sync-settings#384]
[Enter steps to reproduce:]

1. Installed Atom
2. Installed sync-settings package
3. Filled Gist ID and Personal Access Token
4. Entered "Restore" command through Command Palette

**Atom**: 1.21.0 x64
**Electron**: 1.6.9
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [sync-settings](https://github.com/atom-community/sync-settings) package 0.8.3

### Stack Trace

Uncaught SyntaxError: Unexpected token } in JSON at position 287

```
At file:///C:/Users/TheNowak/AppData/Local/atom/app-1.21.0/resources/app.asar/static/index.html:9

SyntaxError: Unexpected token } in JSON at position 287
    at JSON.parse (<anonymous>)
    at /packages/sync-settings/lib/sync-settings.coffee:231:37
    at /packages/sync-settings/node_modules/github/lib/index.js:819:17
    at callCallback (/packages/sync-settings/node_modules/github/lib/index.js:667:17)
    at /packages/sync-settings/node_modules/github/lib/index.js:726:25)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

### Commands

```
     -5:41.8.0 core:confirm (input.hidden-input)
  2x -4:05.2.0 core:paste (input.hidden-input)
     -3:47.5.0 command-palette:toggle (div.package-detail.panels-item)
  2x -3:44.2.0 core:move-down (input.hidden-input)
     -3:42.8.0 core:confirm (input.hidden-input)
     -3:42.8.0 sync-settings:restore (div.package-detail.panels-item)
     -3:13.9.0 command-palette:toggle (div.panels-item)
  2x -3:10.7.0 core:move-down (input.hidden-input)
     -3:09.9.0 core:confirm (input.hidden-input)
     -3:09.9.0 sync-settings:restore (div.panels-item)
     -1:40.3.0 application:open-file (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
```

### Non-Core Packages

```
sync-settings 0.8.3 
```

**Comment by SalathielGenese** [comment]
I got the same error but from CLI, trying to run jasmine on TS using ts-node

**Comment by dirk-thomas** [comment]
Fixed by #422.
